bower.json

```diff
@@ -1,39 +1,39 @@
 {
-    "name": "uber",
-    "version": "14.0.0",
-    "homepage": "https://github.com/magfest/ubersystem",
-    "authors": [
-        "Eli Courtwright <eli@courtwright.org>"
-    ],
-    "description": "MAGFest Ubersystem",
-    "main": "uber/templates/base.html",
-    "license": "BSD",
-    "ignore": [
-        "**/.*",
-        "node_modules",
-        "bower_components",
-        "test",
-        "tests"
-    ],
-    "dependencies": {
-        "lodash": "3.8.0",
-        "jquery": "2.1.4",
-        "jquery-ui": "1.11.4",
-        "bootstrap": "3.2.0",
-        "moment": "2.10.2",
-        "jquery-hoverIntent": "1.8.1",
-        "select2": "4.0.0",
-        "jQuery-contextMenu": "1.6.6",
-        "jquery-confirm": "2.3.1",
-        "toastr": "2.1.1",
-        "footable": "2.0.3",
-        "geocomplete": "1.6.5",
-        "angular": "1.3.15",
-        "angular-route": "1.3.15",
-        "angular-bootstrap": "0.13.0"
-    },
-    "resolutions": {
-        "jquery": ">= 1.9.0",
-        "bootstrap": "3.2.0"
-    }
+  "name": "uber",
+  "version": "14.0.0",
+  "homepage": "https://github.com/magfest/ubersystem",
+  "authors": [
+    "Eli Courtwright <eli@courtwright.org>"
+  ],
+  "description": "MAGFest Ubersystem",
+  "main": "uber/templates/base.html",
+  "license": "BSD",
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests"
+  ],
+  "dependencies": {
+    "lodash": "3.8.0",
+    "jquery": "2.1.4",
+    "jquery-ui": "1.11.4",
+    "bootstrap": "3.2.0",
+    "moment": "2.10.2",
+    "jquery-hoverIntent": "1.8.1",
+    "select2": "4.0.0",
+    "jQuery-contextMenu": "1.6.6",
+    "jquery-confirm": "2.3.1",
+    "toastr": "2.1.1",
+    "geocomplete": "1.6.5",
+    "angular": "1.3.15",
+    "angular-route": "1.3.15",
+    "angular-bootstrap": "0.13.0",
+    "datatables": "~1.10.8"
```

Comment:
You should remove the ``~`` here.  What that means is "Use 1.10.8 or any later version in the 1.10.x line."  Which means that the next time someone adds a new dependency and runs bower, they could literally end up upgrading the version of ``datatables`` they're using without even realizing it.  This is generally undesirable, so we should pin our dependencies to the exact version whenever possible.  (Ideally we'd be using ``npm-shrinkwrap`` to also pin the dependencies of our dependencies, but that's a different ticket.)